include/iceflow/content-type-value.hpp

```diff
@@ -32,7 +32,8 @@ namespace iceflow {
 enum ContentTypeValue : uint32_t {
   UpdateManifest = 128,
   FrameManifest = 129,
-  MainData = 130,        ///< Main data
+  //  MainData = 130,        ///< Main data
+  Manifest = 130,
```

Comment:
I would suggest removing the commented-out duplicate and expanding the documentation on the corrected value.

```suggestion
  Manifest = 130,
```